src/devices/Interop/Windows/WinUser.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
```

Comment:
another one with missing header. as a side not, haven't read the reminder of code yet but is this actually needed?

Reply:
Yes, this is required for the "KeyboardGpioDriver", which is a nice driver for tests on standard PC hardware (the functions are used to blink the keyboard leds)